### PR TITLE
Fix typo in storage developer docs (French)

### PR DESCRIPTION
**Pull request by graz-network**
Typo and negativ form uncomplete
 
-- Typo : "dur" changed in "sur"

-- "Ethereum n'a été conçu pour cela"  changed by : "Ethereum n'a pas été conçu pour cela"

### CODE REVIEW

**Review by SuperDelphi — CHANGES_REQUESTED** [review]
It's almost all good. Since we agreed upon the usage of "gaz" instead of "carburant", we could change the "carburant" occurrence here. (SuperDelphi#2468 on Discord)

File: src/content/translations/fr/developers/docs/storage/index.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ lang: fr
...
+Ethereum lui-même peut être utilisé comme un système de stockage décentralisé, c'est d'ailleurs déjà le cas concernant le stockage de code compris dans tous les contrats intelligent. Cependant, lorsqu'il s'agit de grandes quantités de données, Ethereum n'a pas été conçu pour cela. La chaîne ne cesse de croître, mais au moment d'écrire ces lignes, la chaîne Ethereum est d'environ 500 Go à 1 To ([selon le client](https://etherscan.io/chartsync/chaindefault)), et chaque nœud du réseau doit être en mesure de stocker toutes les données. Si la chaîne devait s'étendre à de grandes quantités de données (disons 5 To par exemple), il serait impossible pour tous les nœuds de continuer à fonctionner. En outre, le coût du déploiement d'une telle quantité de données sur le réseau principal serait prohibitif en raison des frais de [carburant](/developers/docs/gas).
```
Comment (minimalsm) [inline src/content/translations/fr/developers/docs/storage/index.md:10]:
```suggestion
Ethereum lui-même peut être utilisé comme un système de stockage décentralisé, c'est d'ailleurs déjà le cas concernant le stockage de code compris dans tous les contrats intelligent. Cependant, lorsqu'il s'agit de grandes quantités de données, Ethereum n'a pas été conçu pour cela. La chaîne ne cesse de croître, mais au moment d'écrire ces lignes, la chaîne Ethereum est d'environ 500 Go à 1 To ([selon le client](https://etherscan.io/chartsync/chaindefault)), et chaque nœud du réseau doit être en mesure de stocker toutes les données. Si la chaîne devait s'étendre à de grandes quantités de données (disons 5 To par exemple), il serait impossible pour tous les nœuds de continuer à fonctionner. En outre, le coût du déploiement d'une telle quantité de données sur le réseau principal serait prohibitif en raison des frais de [gaz](/developers/docs/gas).
```

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Thanks @graz-network 

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @graz-network for docs, translation

**Pull request merged**